Makefile

```diff
@@ -221,3 +209,14 @@ $(DEBIANDIR)/% : debian/%
 	mkdir -p $(DEBIANDIR)
 	cp -f $< $@
 
+#### FORTRAN BINDINGS ####
+
+export NCCL_MAJOR NCCL_MINOR NCCL_PATCH CUDA_MAJOR CUDA_MINOR LIBLINK
+
+forlib : lib
+	$(MAKE) -C fortran lib BUILDDIR=../$(BUILDDIR)
```

Comment:
Right. I wanted to avoid the absolute path as it makes the log ugly, but that doesn't work. I need to figure out another way or fall back on absolute path.